docs/quiz-toolkit/overview.md

```diff
@@ -0,0 +1,21 @@
+---
+sidebar_label: "Overview"
+sidebar_position: 1
+---
+# Overview
+
+
+**MecSimCalc** now supports online quizzes, allowing professors to efficiently administer quizzes and automatically collect data. This new functionality enriches MecSimCalc's capabilities, enabling dynamic input generation where quiz values can be **randomized**, and **correctness is determined through logical code evaluation**.
+
+Currently, MecSimCalc offers two convenient methods for data collection:
+
+## [Google Sheets](google-sheet)
```

Comment:
Have a "Steps" title and then have steps such as
1. Create your quiz App
2. Store results in Google Sheets
3. Send an automated email to yourself and students when they submit.

We basically want to help them create an entire app. Maybe also show a demo Engg 130 quiz app? Or a demo app somewhere. Then they can click a link and create a new one from a template.